UnityProject/Assets/Scripts/Objects/PushPull.cs

```diff
@@ -39,6 +39,12 @@ public PushPull parentContainer
 		get => _parentContainer;
 		set
 		{
+			if(value == null)
+			{
+				Logger.LogError(gameObject.name + " tried to set parentContainer to null!", Category.Transform);
```

Comment:
parentContainer is deliberately set to null in almost a dozen situations. Might cause issues doing this.

Reply:
Oh, I must have missed it somehow. I removed it in the last commit.